composer.json

```diff
@@ -29,7 +29,10 @@
   "require-dev": {
     "phpspec/phpspec": "~2.1",
     "doctrine/orm": "~2.3",
-    "phpdocumentor/reflection": "~1.0"
+    "phpdocumentor/reflection": "~1.0.3"
+  },
+  "conflict": {
```

Comment:
I don't think so because it is not a required package, only for dev.

If user include this one, the `conflict` option will do the work, not the `require-dev`.

The more simple and reliable is to keep `"phpdocumentor/reflection": "~1.0"` on the require-dev and adjust patch version with `conflict`